docs/user/scaling.rst

```diff
@@ -0,0 +1,43 @@
+.. _scaling:
+
+Scaling
+=======
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+
+Artemis is a scalable system that supports large courses: Some courses at TUM have been conducted using Artemis with more than 2000 students.
+Depending on your setup and the number of user - as well as other requirements such as availability - it might be necessary for you to scale your Artemis installations.
```

Comment:
```suggestion
Depending on your setup and the number of users - as well as other requirements such as availability - it might be necessary for you to scale your Artemis installations.
```